docs/src/guides/inscriptions.md

```diff
@@ -58,6 +58,12 @@ index, add the following to your `bitcoin.conf`:
 
 ```
 txindex=1
+
+#manual connections for relaying transactions without any issues
+addnode=full-rbf1.btc.petertodd.net
+addnode=full-rbf2.btc.petertodd.net
+addnode=full-rbf3.btc.petertodd.net
+addnode=full-rbf4.btc.petertodd.net
```

Comment:
```suggestion
addnode=full-rbf4.btc.petertodd.net
mempoolfullrbf=1
```

If you're going to connect to these full-rbf nodes, at least have the decency to also turn on full-rbf so they're not wasting connection slots.

Reply:
Added it in the last commit